### ISSUE TITLE
[Feature] Migration/mirror from subversion (svn)

### ISSUE THREAD

**Issue by askovpen** [go-gitea/gitea#10095]
feature request: add migration/mirror from subversion - like 'git svn' command.

**Comment by lunny** [comment]
I think this is a duplicated issue.

**Comment by askovpen** [comment]
@lunny I tried to find before create new issue. it is first.

**Comment by lunny** [comment]
#4158

**Comment by askovpen** [comment]
yes. closed :-(

**Comment by lafriks** [comment]
Until there is good svn to git migration library available written in golang I don't see how this could possibly be added to Gitea so I propose to closing it as I don't see that happening anytime soon if ever

**Comment by davidsvantesson** [comment]
I work for a organization that migrated from svn to git. Svn is quite different structure from Git and there are no straigtforward way to migrate. First it should be noted that only the repo itself can be  migrated; no issues, pull requests or releases, as that is completly organizaiton dependent and dependent on other tools. Then the repo migration will depend completely on the organizaiton svn setup as well, like branch and tag setup. There are some helper tools available to do migrations (e.g. git svn) but you need to write a script for your setup. Once that is done it is quite simple to use gitea's API to get the migration in the way you want. That assumes you have a lot of svn repositories, otherwise it is just easier to do the migration "by hand".
So I agree there should not be a Gitea issue for this.

**Comment by zeripath** [comment]
You can run git-as-svn around Gitea too.
